$NetBSD: patch-ai,v 1.3 2010/06/12 19:36:01 dholland Exp $

Print and parse time_t as intmax_t, not as long (which might not fit).
Likewise for (perhaps only some uses of) off_t.

The last hunk is unrelated to the rest and fixes a tilde expansion bug.

--- src/server.c.orig	1999-08-04 15:57:33.000000000 +0000
+++ src/server.c
@@ -1737,7 +1753,7 @@ static void settarget(cmd, isdir)
 	/*
 	 * Handle target
 	 */
-	if (exptilde(target, cp) == NULL)
+	if (exptilde(target, file) == NULL)
 		return;
 	ptarget = target;
 	while (*ptarget)
